koku/reporting/migrations/0290_update_settings.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.18 on 2023-05-22 20:38
+from django.db import migrations
+
+
+def update_settings(apps, schema_editor):
+    UserSettings = apps.get_model("reporting", "UserSettings")
+    if user_settings := UserSettings.objects.first():
+        if user_settings.settings.get("cost_type") == "savingsplan_effective_cost":
+            user_settings.settings["cost_type"] = "calculated_amortized_cost"
+            user_settings.save()
+
+
+def revert(apps, schema_editor):
+    UserSettings = apps.get_model("reporting", "UserSettings")
+    if user_settings := UserSettings.objects.first():
```

Comment:
You may need to refresh my mind about how user settings will work, but will there only ever be one row for each schema?

Reply:
Correct. It's just one row and the `settings` column is a JSON field that contains the settings values.